### ISSUE TITLE
Esconder codígo de resolução

### ISSUE THREAD

**Issue by AlissonSteffens** [UNIVALI-LITE/vagas#3]
Não tá muito exposta a solução no <code>wellcometobrawlhalla.js</code>?

**Comment by AlissonSteffens** [comment]
Uma solução pode ser um webservice no heroku

**Comment by noschang** [comment]
Não. Eu estou deixando assim por enquanto. Depois que estiver pronto e testado eu vou obfuscar com o Closure da Google.
